src/frontend/src/styles/main.css

```diff
@@ -949,6 +961,159 @@ a:hover,
   color: rgba(0, 0, 0, 0.5);
 }
 
+/**
+ * LandingPage component
+ */
+
+.c-landingPage {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--rs-main-content-width), 1fr);
+  grid-template-areas: "left right";
+
+  height: 100vh;
+}
+
+.c-landingPage__logo {
+  position: absolute;
+  left: var(--rs-landing-page-left-pane-padding);
+  top: var(--rs-landing-page-left-pane-padding);
+
+  color: var(--rc-landing-page-background-text);
+}
+
+.c-landingPage__logo .c-logo {
+  padding: 0;
+}
+
+.c-landingPage__left {
+  background: var(--rc-landing-page-background);
+  color: var(--rc-landing-page-background-text);
+
+  grid-area: left;
+
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+
+  padding: var(--rs-landing-page-left-pane-padding);
+}
+
+.c-landingPage__left__content {
+  flex: 1;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  padding: 0 20%;
+
+  /* To compensate for the IC logo that is outisde the normal flow with position absolute */
+  --logo-height: 3rem;
+  margin-top: var(--logo-height);
+}
+
+.c-landingPage__logo--mobile {
+  display: none;
+}
+
+.c-landingPage__logo--desktop {
+  display: block;
+}
+
+.c-landingPage__right {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  grid-area: right;
+
+  overflow-y: scroll;
+}
+
+.c-landingPage__right__content {
+  min-width: var(--rs-main-content-width);
+  max-width: var(--rs-main-content-width);
+  /* This is necessary in case the content is bigger than the parent. */
+  /* It keeps the start of the box at the top of the container. */
+  max-height: 90%;
+
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+.c-landingPage__right__content--full-width {
+  width: 100%;
+}
+
+.c-landingPage .c-footer__link {
+  padding: 0;
+}
+
+.c-landingPage__left__footer {
+  display: flex;
+  gap: var(--rs-inline-grid-gap);
+}
+
+@media (max-width: 512px) {
+  .c-landingPage {
+    display: block;
+
+    padding: 0 var(--rs-landing-page-padding--mobile);
+
+    background: var(--rc-landing-page-background);
+
+    height: auto;
+    min-height: 100vh;
+  }
+
+  .c-landingPage__logo {
+    display: flex;
+    position: static;
```

Comment:
why position static? its the default anyway

Reply:
Because I set it to `absolute` for desktop and I want the default for mobile.